plugins/scaffolder-backend-module-gitlab/src/actions/createGitlabProjectAccessTokenAction.ts

```diff
@@ -35,46 +41,86 @@ export const createGitlabProjectAccessTokenAction = (options: {
     id: 'gitlab:projectAccessToken:create',
     examples,
     schema: {
-      input: commonGitlabConfig.merge(
```

Comment:
Removing this could be a breaking change. It's hard to see right now until typescript passes and we can build the api-reports.